src/firefly/js/ui/FieldGroup.jsx

```diff
@@ -19,6 +36,7 @@ export const FieldGroup = memo( ({keepMounted, reducerFunc=undefined, groupKey,
         unregister:wrapperUnregister,
         registeredComponents:wrapperRegisteredComponents}= useContext(FieldGroupCtx);
 
+
```

Comment:
Forcing a FieldGroup re-render when `registeredComponents` changes seem problematic.  Imagine having a complex page/form with many components with only one small component uses this feature, it will trigger the whole page/form to re-render.  Something to be aware of if there is regression issues in the future.

Reply:
I will look

Reply:
OK- looked. This was already in place before this PR, and it is being used by another PR so I can't just change it without trying it in the other branch. I can look at it in a week or two after everything gets merge. 

That said, I don't think it is a big deal. Just because `FieldGroup` re-renders it does not mean that any of the children re-render. In fact, they almost never do because it does not affect their props or state.